src/skeleton/Blocks.scala

```diff
@@ -9,6 +9,7 @@ import freechips.rocketchip.subsystem._
 import freechips.rocketchip.diplomaticobjectmodel.logicaltree.LogicalTreeNode
 
 case class BlockAttachParams(
+  sbus: TLBusWrapper,	
```

Comment:
This will be a backwards-incompatible change, since it introduces a new, non-optional argument. A way to make this backwards compatible would be to introduce it as an `Option` type and have it default to `None`.

However, before making that change, could @albertchen-sifive, @rmac-sifive, @erikdanie, and @hcook comment on how the recent peripherals refactoring may affect how these `BlockAttachParams` are specified? Should we update the SkeletonDUT to make use of the new [`Attachable`](https://github.com/chipsalliance/rocket-chip/blob/27120ee42ff4ff41908981025efcf7d59f879894/src/main/scala/subsystem/BaseSubsystem.scala#L48-L59) trait?

Reply:
I'm not sure the answer to this, but we may need to merge this soon, because when I tried to bump federation in PCS master it errored because this wasn't merged in

Reply:
@erikdanie I need to investigate. soc-testsocket should be used only for HCA standalone simulations, not in a federation/PCS context.